src/components/Content/index.tsx

```diff
@@ -26,7 +26,7 @@ export const Content = () => {
   }
 
   return (
-    <Layout>
+    <PageLayout style={{ background: "white" }}>
```

Comment:
I see that PageLayout already has ${whiteBackground} which has the background style as white. Why are we using inline styles here to set background?

Reply:
must have missed this one. will delete